docs/kyma/docs/037-inst-custom-istio.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Installation with custom Istio deployment
+type: Installation
+---
+
+You can use Kyma with a custom deployment of Istio that you installed in the target environment. To enable such implementation, remove Istio from the list of components that install with Kyma.
+The version of your Istio deployment must match the version that Kyma currently supports.
+
+In the installation process, the installer applies a custom patch to every Istio deployment. This is a mandatory step.  
+
+>**NOTE:** To learn more, read the **Istio patch** document in the **Service Mesh** documentation topic.
+
+## Prerequisites
+
+- A live Istio version compatible with the version currently supported by Kyma. To check the supported version, see the `resources/istio-kyma-patch/templates/job.yaml` file.
```

Comment:
Please provide a link to istio installation guide.

Reply:
Added